### PR TITLE
Move Bing Search SDKs from Preview to GA

### PR DESCRIPTION
**Pull request by sudhansu88**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj [hunk 1]
```diff
@@ -3,10 +3,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.SpellCheck</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services SpellCheck API.</Description>
-    <VersionPrefix>0.18.0-preview</VersionPrefix>
+    <VersionPrefix>1.1.0</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj]: The stable version should start from `1.0.0` here

File: src/SDKs/CognitiveServices/dataPlane/Search/BingCustomSearch/BingCustomSearch/Microsoft.Azure.CognitiveServices.Search.CustomSearch.csproj [hunk 2]
```diff
@@ -6,10 +6,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Search.CustomSearch</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Custom Search API.</Description>
-    <VersionPrefix>1.2.0-preview</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Search/BingCustomSearch/BingCustomSearch/Microsoft.Azure.CognitiveServices.Search.CustomSearch.csproj]: The stable version should be at least equivalent to the preview version and lower than the major version
You should ideally have the stable version as `1.2.0` here

File: src/SDKs/CognitiveServices/dataPlane/Search/BingCustomSearch/BingCustomSearch/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Custom Search Client Library")]
 [assembly: AssemblyDescription("Provides API functions for consuming the Microsoft Azure Cognitive Services Custom Search API.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Search/BingCustomSearch/BingCustomSearch/Properties/AssemblyInfo.cs]: Please do not bump the `AssemblyVersion` here however the `AssemblyFileVersion` must be updated to `1.2.0`

File: src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services SpellCheck Client Library")]
 [assembly: AssemblyDescription("Provides API functions for consuming the Microsoft Azure Cognitive Services SpellCheck API.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Properties/AssemblyInfo.cs]: Please do not update the `AssemblyVersion` here

File: src/SDKs/CognitiveServices/dataPlane/Search/BingEntitySearch/BingEntitySearch/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Entity Search Client Library")]
 [assembly: AssemblyDescription("Provides API functions for consuming the Microsoft Azure Cognitive Services Entity Search API.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Search/BingEntitySearch/BingEntitySearch/Properties/AssemblyInfo.cs]: Please do not update the `AssemblyVersion` here

File: src/SDKs/CognitiveServices/dataPlane/Search/BingImageSearch/BingImageSearch/Properties/AssemblyInfo.cs [hunk 6]
```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Image Search Client Library")]
 [assembly: AssemblyDescription("Provides API functions for consuming the Microsoft Azure Cognitive Services Image Search API.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Search/BingImageSearch/BingImageSearch/Properties/AssemblyInfo.cs]: Please do not update the AssemblyVersion here

File: src/SDKs/CognitiveServices/dataPlane/Search/BingNewsSearch/BingNewsSearch/Properties/AssemblyInfo.cs [hunk 7]
```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services News Search Client Library")]
 [assembly: AssemblyDescription("Provides API functions for consuming the Microsoft Azure Cognitive Services News Search API.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Search/BingNewsSearch/BingNewsSearch/Properties/AssemblyInfo.cs]: Please do not update the AssemblyVersion here

File: src/SDKs/CognitiveServices/dataPlane/Search/BingVideoSearch/BingVideoSearch/Properties/AssemblyInfo.cs [hunk 8]
```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Web Search Client Library")]
 [assembly: AssemblyDescription("Provides API functions for consuming the Microsoft Azure Cognitive Services Video Search API.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Search/BingVideoSearch/BingVideoSearch/Properties/AssemblyInfo.cs]: Please do not update the AssemblyVersion here

File: src/SDKs/CognitiveServices/dataPlane/Search/BingWebSearch/BingWebSearch/Properties/AssemblyInfo.cs [hunk 9]
```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Web Search Client Library")]
 [assembly: AssemblyDescription("Provides API functions for consuming the Microsoft Azure Cognitive Services Web Search API.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Search/BingWebSearch/BingWebSearch/Properties/AssemblyInfo.cs]: Please do not update the AssemblyVersion here

**Review by dsgouda — CHANGES_REQUESTED** [review]
Version numbers should according to nuget conventions
Alternatively, you can bump the version numbers across all packages to `1.2.0` here

@ronak25 Please review the PR and verify the changes suggested

File: src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj (hunk 1 above)
Comment (sudhansu88) [reply src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj]: Sure

File: src/SDKs/CognitiveServices/dataPlane/Search/BingCustomSearch/BingCustomSearch/Microsoft.Azure.CognitiveServices.Search.CustomSearch.csproj (hunk 2 above)
Comment (sudhansu88) [reply src/SDKs/CognitiveServices/dataPlane/Search/BingCustomSearch/BingCustomSearch/Microsoft.Azure.CognitiveServices.Search.CustomSearch.csproj]: Made all stable versions as 1.2.0

File: src/SDKs/CognitiveServices/dataPlane/Search/BingEntitySearch/BingEntitySearch/Properties/AssemblyInfo.cs (hunk 5 above)
Comment (sudhansu88) [reply src/SDKs/CognitiveServices/dataPlane/Search/BingEntitySearch/BingEntitySearch/Properties/AssemblyInfo.cs]: Undone

**Comment by sudhansu88** [comment]
All well ? Please let me know if we are waiting for anything else.

**Comment by dsgouda** [comment]
@sudhansu88 we need a review from @ronak25 to proceed

**Review by ronak25 — APPROVED** [review]
Changes look good for GA.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**